ddev/src/ddev/cli/release/list_versions/__init__.py

```diff
@@ -0,0 +1,39 @@
+# (C) Datadog, Inc. 2023-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+import click
+
+if TYPE_CHECKING:
+    from ddev.cli.application import Application
+
+
+@click.command("list", short_help='Show all versions of an integration')
+@click.argument('integration')
+@click.pass_context
+def list_versions(ctx: click.Context, integration: str):
+    """Show all versions of an integration."""
+    import httpx
+    from packaging.version import Version
+
+    integration = integration.replace('_', '-')
+    ignored_prefix = 'datadog-'
+    if integration.startswith(ignored_prefix):
+        integration = integration[len(ignored_prefix) :]
+
+    integration_url = f'https://dd-integrations-core-wheels-build-stable.datadoghq.com/targets/simple/datadog-{integration}/index.html'
+
+    response = httpx.get(integration_url)
+    versions = response.text.splitlines()
+
+    version_numbers = []
+    for line in versions:
+        version_number = line.split('-')[1]
+        version_numbers.append(Version(version_number))
+
+    version_numbers.sort()
+    for ver in version_numbers:
+        print(str(ver))
```

Comment:
For displaying text we actually have methods that we use to enforce colors, text that must always be displayed can use https://github.com/DataDog/integrations-core/blob/50ae2bffedbb8a7f56be8d4d26872f5304eb5dfa/ddev/src/ddev/cli/terminal.py#L61-L62

```suggestion
        print(app.display(str(ver)))
```